dotnet/samples/KernelSyntaxExamples/Example58_FunctionCalling.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Kusto.Cloud.Platform.Utils;
```

Comment:
This namespace is used for `string.IsNotNullOrEmpty()` extension method in line 68, but I would avoid using extension methods from other libraries in this example and just use `!string.IsNullOrEmpty(chatMessage.Content)`. In this case it will be easier for users to copy this example with minimal changes required.

Reply:
Oh, that is a mistake. I think that was Visual Studio trying (but failing) to be helpful. Will change!